cns/azure-cns.yaml

```diff
@@ -127,9 +156,9 @@ spec:
               value: /etc/azure-cns/cns_config.json
             - name: NODENAME
               valueFrom:
-                  fieldRef:
```

Comment:
not sure if we want to add the new *CIDR environment variables here on the reference yaml template - since they're specific to swiftv2 scenario.